### DISCUSSION TITLE
Failing test - pyForceFieldConstraints x testUFFAngleConstraints

### DISCUSSION THREAD

**Discussion by pavelToman** [rdkit/rdkit#7588] [Q&A] [answered]
Hello,
I am trying to install and provide RDKit to researchers on UGent HPC cluster. We are using EasyBuild to handle software - I need to build it from source.
Build and installation seems fine, but there is a problem with one test - it is failing but not on all the nodes, only on one.
Test is ok on node:
- RHEL 8.8 (Linux 4.18.0-477.43.1.el8_8.x86_64, python 3.6.8)
- Rocky Linux 8.9, x86_64, Intel(R) Xeon(R) CPU E5-2667 v3 @ 3.20GHz (haswell), Python 3.6.8

But it is failing on:
- Rocky Linux 9.4, x86_64, AMD EPYC-Milan Processor (zen3), Python 3.9.18
- LOG: https://github.com/easybuilders/easybuild-easyconfigs/pull/20824#issuecomment-2208588403 
- it is a little bit similar with https://github.com/rdkit/rdkit/issues/5252

Could you help me with this please?

**Comment by greglandrum** [comment, marked as answer]
@pavelToman It looks like the failure is due to a numerical problem in the minimizer. I don't have access to a machine with that CPU to play around and do some testing, so I can't be directly helpful. Still: given that all the other tests passed, this one would not make me too nervous and I'd be happy deploying the build as-is (i.e. ignoring that failure).

Out of curiosity: which compiler are you using and what are your CXXFLAGS?

**Reply by pavelToman** [reply]
Thank you for your answer @greglandrum !
The compiler is GCC 12.3.0 + FlexiBLAS (incl. LAPACK) + FFTW + OpenMPI + ScaLAPACK.
The CXXFLAGS='-O2 -ftree-vectorize -march=native -fno-math-errno'

**Reply by ThomasHoffmann77** [reply]
@greglandrum Looks like Minimize did not converge within 200 steps. The test passes, if I change 
`r = ff.Minimize()`
to 
`r = ff.Minimize(500)`
Would  this be an appropriate solution?
